src/server/scripts/Spells/spell_item.cpp

```diff
@@ -865,6 +865,52 @@ class spell_item_gnomish_universal_remote : public SpellScriptLoader
     }
 };
 
+class spell_item_poweful_anti_venom : public SpellScriptLoader
+{
+public:
+    spell_item_poweful_anti_venom() : SpellScriptLoader("spell_item_poweful_anti_venom") {}
+
+    class spell_item_powerful_anti_venom_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_item_powerful_anti_venom_SpellScript);
+
+        void HandleDummy(SpellEffIndex effIndex)
+        {
+            PreventHitDefaultEffect(effIndex);
+            if (Unit* target = GetHitUnit())
+            {
+                std::list<uint32> removeList;
+                Unit::AuraMap const& auras = target->GetOwnedAuras();
+                for (Unit::AuraMap::const_iterator itr = auras.begin(); itr != auras.end(); ++itr)
+                {
+                    Aura* aura = itr->second;
+                    if (aura->GetSpellInfo()->SpellLevel > 60 || aura->GetSpellInfo()->Dispel != DISPEL_POISON)
+                    {
```

Comment:
Could it make more sense to use the following snippet?  Seems easier to read than a negative, but maybe it breaks some guidelines that I have not seen.
```
if(aura->GetSpellInfo()->SpellLevel <= 60 && aura->GetSpellInfo()->Dispel == DISPEL_POISON)
{
    removeList.push_back(aura->GetId());
}
```

Reply:
No difference.